tests/integration/test_design.py

```diff
@@ -505,12 +505,15 @@ def test_named_selections(modeler: Modeler):
     assert len(design.named_selections) == 3
 
     # Try to vefify name selection using earlier backend version
-    design = modeler.open_file(Path(FILES_DIR, "25R1BasicBoxNameSelection.scdocx"))
     oldbackend = design._grpc_client._backend_version
-    design._grpc_client._backend_version = (24, 2, 0)
-    hello = design.named_selections
-    assert hello[0].faces == []
-    design._grpc_client._backend_version = oldbackend
+    try:
+        design = modeler.open_file(Path(FILES_DIR, "25R1BasicBoxNameSelection.scdocx"))
+        design._grpc_client._backend_version = (24, 2, 0)
+        hello = design.named_selections
+        assert hello[0].faces == []
+        design._grpc_client._backend_version = oldbackend
+    except:
+        design._grpc_client._backend_version = oldbackend
```

Comment:
```suggestion
    except Exception as err:
        raise err
    finally:
        design._grpc_client._backend_version = oldbackend
```
Although you might want to move to the fixture approach based on offline conversations =)